.gitignore

```diff
@@ -3,6 +3,7 @@
 .ruby-version
 test.sass
 npm-debug.log
+/docs/_config.local.yml
```

Comment:
It's already in the gitignore 😉  https://github.com/jgthms/bulma/blob/master/docs/.gitignore